app/views/recipes/_modal.html.erb

```diff
@@ -0,0 +1,18 @@
+<!-- _modal.html.erb -->
+<div class="modal" id="myModal">
+  <div class="modal-content">
+    <span class="close" id="closeModalButton">&times;</span>
+    <h3 class="modal-title">Generate Shopping List</h3>
+    <%= form_with(url: shopping_list_path(recipe_id: @recipe.id), method: :get) do |form| %>
+    <div class="form-group">
+         <%= form.hidden_field :recipe_id, value: @recipe.id %>
+        <%= form.label :inventory_id, "Select Inventory" %>
+        <%= form.select :inventory_id, options_for_select(@inventories.map { |inventory| [inventory.name, inventory.id] }), {}, class: 'form-control' %>
+
+    </div>
+  <div class="form-group">
+    <%= form.submit "Generate Shopping List", class: "btn btn-primary", id: "generate-shopping-list" %>
+  </div>
+<% end %>
+  </div>
+</div>

```

Comment:
- [ ] Please style the modal just like in the wireframe shown below 👇 
### Wireframe Modal
![image](https://github.com/monicah79/Recipe-app-Group-project/assets/46839548/15929ba4-187e-47f0-b46e-d7873b517167)

### Your modal design
![image](https://github.com/monicah79/Recipe-app-Group-project/assets/46839548/1b35f673-a4ee-4ef1-9ed7-6a2c07f77cd0)